src/views/Expenses.js

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useState } from 'react';
+import AddExpensesForm from '../components/AddExpensesForm';
+import ExpensesList from '../components/ExpensesList';
+
+const Expenses = () => {
+  const [expenses, setExpenses] = useState([]);
+  const token = process.env.REACT_APP_MOSTASH_API_KEY;
+  const baseURL = process.env.REACT_APP_MOSTASH_BASE_URL;
+  const headers = {
+    'Stash': token,
+    'Content-Type': 'application/json'
+  };
+
+  // Run on initial render only
+  useEffect(() => {
+    const fetchExpenses = async () => {
+      const url = `${baseURL}/items.json?stash=${token}&kind=expense`;
+
+      try {
+        const response = await fetch(url);
+        const expenseData = await response.json();
+        setExpenses(expenseData);
+      } catch {
+        alert('Error loading expenses. Please try again later.');
```

Comment:
there is a bit of duplication here. maybe in the next PR we can add a method which does the fetch request, parses the json and handles the error.

Reply:
Ah okay, so like a single function that handles all the request types?

Reply:
yeah, it is just an idea. maybe something with a signature like (just to illustrate what I mean) `sendRequest(relativeUrl, method, body)`

Reply:
Gotcha, I'll look into it.